src/hwdb/hwdb.c

```diff
@@ -61,6 +62,7 @@ static int help(void) {
                "     --version    Show package version\n"
                "  -s --strict     When updating, return non-zero exit value on any parsing error\n"
                "     --usr        Generate in " UDEVLIBEXECDIR " instead of /etc/udev\n"
+               "     --compat     Generate database in old format\n"
```

Comment:
`s/old format/old format read by udevadm/` ?

This also needs a man page entry, with some guidance when the `--compat` format should be used.